sensu-plugins-elasticsearch.gemspec

```diff
@@ -42,6 +42,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'elasticsearch',     '1.0.14'
   s.add_runtime_dependency 'aws-es-transport',  '~> 0.1'
   s.add_runtime_dependency 'sensu-plugin',      '~> 1.2'
+  s.add_runtime_dependency 'json',      '~> 1.8'
```

Comment:
when I ran bundle I got it wanting to install json 2.x which is not compatible with the latest version of httparty: https://rubygems.org/gems/httparty